installing/installing_with_agent_based_installer/prepare-pxe-assets-agent.adoc

```diff
@@ -41,10 +41,21 @@ include::modules/installing-ocp-agent-ibm-z.adoc[leveloffset=+1]
 // Adding {ibm-z-title} agents with z/VM
 include::modules/installing-ocp-agent-ibm-z-zvm.adoc[leveloffset=+2]
 
-[role="_additional-resources"]
+include::modules/adding-ibmz-lpar-agent.adoc[leveloffset=+2]
+
```

Comment:
You're including the module twice in this assembly, and it's probably why the builds are failing. I would remove this instance here, and add back the role that was deleted (either way, that role needs to be put back):
```suggestion
[role="_additional-resources"]
```